tests/functions/polynomial.cpp

```diff
@@ -55,15 +55,15 @@ TEST_CASE("Polynomial evaluation", "[poly_evalf], [polynomials]") {
     Polynomial P(c, &a, &b);
 
     SECTION("Evaluation within bounds") {
-        double x = 1.5;
-        double calc_val = P.evalf(&x);
-        double ref_val = 1.0 + 2.0*x*x;
+        Coord<1> x = {1.5};
```

Comment:
I think `Coord<1> x{1.5};` might achieve the same effect.

Reply:
Hmm, I found this version better than `x({1.5})` which I had first, but if it works without the `()`, then I agree

Reply:
It should work _without_ `()` there are some subtleties for copying _vs_ initializing, but it shouldn't matter at all in this case.